projects/1.20/assets/tacz-durability/gundurability/lang/zh_cn.json

```
@@ -0,0 +1,25 @@
+{
+  "key.gundurability.inspect_durability": "退弹（默认设置为Inspect键）",
+  "gamerule.jamChance": "卡壳概率",
+  "key.gundurability.durability": "检查耐久度",
+  "item.gundurability.brass_brush": "铜刷",
+  "item.gundurability.wd_40": "WD 40（枪油）",
+  "gui.gundurability.repair_gui.tooltip_brush": "刷",
+  "effect.gundurability.checkingdurability": "检查耐久度",
+  "gamerule.jamChance.description": "枪卡壳的概率为多少，默认25",
+  "gui.gundurability.repair_gui.tooltip_wd40": "枪油",
+  "gamerule.gunbreak": "枪支损坏",
+  "gamerule.gunbreak.description": "让枪在耐久度为0时损坏",
+  "block.gundurability.repair_table": "修理台",
+  "attribute.gundurability.handling": "处理枪支",
+  "gui.gundurability.repair_gui.button_apply_spraybrush": "修复",
+  "item_group.gundurability.tacz_durability": "TACZ-耐久附属",
```

```suggestion
  "item_group.gundurability.tacz_durability": "TACZ 耐久附属",
```
除非其他的TACZ附属都加连接符，否则按原文为准